### PR TITLE
Us 24 disable editing for previous weeks

### PR DESCRIPTION
**Pull request by dawidk-kainos**

### CODE REVIEW

**Comment by r-kruk-kainos** [comment]
Code review done :)

File: src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js [hunk 1]
```diff
@@ -65,19 +67,26 @@ define(['angular', 'application/interviewer/tdprInterviewerModule', 'application
...
             $scope.hasNoteChanged=false;
-        }
+        };
 
         $scope.changeSlotStatus = function() {
+            if($scope.relativeDayNumber<0 && !isRecruiter){
+                if(!isNotificationShown) {
+                    Notification.error({message: 'You cannot edit slots from past weeks!', delay: 2000});
+                }
+                isNotificationShown=!isNotificationShown;
```
Comment (rlewan) [inline src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js]: Is the isNotificationShown flag supposed to block multiple error messages from appearing?

Comment (dawidk-kainos) [inline src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js]: Yes

Comment (rlewan) [inline src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js]: As discussed, please try to use only one handler for `changeSlotStatus` - either click or mousedown. Then we can remove the `isNotificationShown` and corresponding if condition.

File: src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js [hunk 2]
```diff
@@ -213,12 +220,6 @@ define(['angular', 'application/interviewer/tdprInterviewerModule', 'application
                }
         )}
 
-        function getDayOfTheWeek(d, i) {
-            var day = d.getDay(),
-                diff = d.getDate() - day + (day == 0 ? -6:1); // adjust when day is sunday
-            return new Date(d.setDate(diff+i)); //i = 0 - monday
-        }
-
```
Comment (rlewan) [inline src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js:221]: Good boy-scouting 👍

**Pull request merged**